parl/remote/worker.py

```diff
@@ -357,9 +361,36 @@ def _reply_heartbeat(self, target):
             "[Worker] lost connection with the master, will exit replying heartbeat for master."
         )
         self.worker_status.clear()
+        self.log_server_proc.kill()
         # exit the worker
         self.worker_is_alive = False
 
+    def _create_log_server(self):
+        log_server_file = __file__.replace('worker.pyc', 'log_server.py')
+        log_server_file = log_server_file.replace('worker.py', 'log_server.py')
+
+        # Get a random port for log server
```

Comment:
Please put `get_free_tcp_port` function of `parl/remote/scripts.py` to the `parl/utils/machine_info.py`, and use that function instead.